devito/core/autotuning.py

```diff
@@ -319,13 +316,14 @@ def generate_block_shapes(blockable, args, level):
         for bs in list(ret):
             handle = []
             for v in options['blocksize-l1']:
-                # To be a valid blocksize, it must be smaller than and divide evenly
-                # the parent's block size
-                if all(v <= i and i % v == 0 for _, i in bs):
+                # To be a valid block size, it must be smaller than
+                # and divide evenly the parent's block size.
+                # Blocksizes equal to the parent's block size are not included
```

Comment:
also why not? it would be sort-of like defaulting to the non-hierarchical case no?

Reply:
I would expect that if someone asks for >1 levels would not be interested in that....

Reply:
as in, why would you remove a bunch of points from the exploration space?